apis/apps/v1alpha1/cluster_types.go

```diff
@@ -1216,6 +1221,27 @@ type ClusterComponentService struct {
 	PodService *bool `json:"podService,omitempty"`
 }
 
+type ComponentSystemAccount struct {
+	// The name of the system account.
+	//
+	// +kubebuilder:validation:Required
+	Name string `json:"name"`
+
```

Comment:
Should the name of the system account be customizable and modifiable as well?
For example, some external database administrator systems may have assumptions or enforce a default value for the name of the superuser account.

Reply:
It can be expanded in the future when there is actual requirements.